packages/opentelemetry/src/utils/getTraceData.ts

```diff
@@ -0,0 +1,29 @@
+import * as api from '@opentelemetry/api';
+import type { SerializedTraceData } from '@sentry/types';
+import { dropUndefinedKeys } from '@sentry/utils';
+
+/**
+ * Otel-specific implementation of `getTraceData`.
+ * @see `@sentry/core` version of `getTraceData` for more information
+ */
+export function getTraceData(): SerializedTraceData {
+  const context = api.context.active();
+
+  // This should never happen, given we always create an ambient non-recording span if there's no active span.
+  if (!context) {
```

Comment:
I _think_ this cannot be empty? 🤔 What does the type say?

Reply:
I could have sworn TS screamed at me previously that the type was `Context | undefined` but after re-checking you're right, we always get back a context. Adjusted it